packages/documentation-site/patternfly-docs/content/developer-resources/accessibility-development.md

```diff
@@ -1,116 +1,125 @@
-- **Is it interactable?** If you can get to it, it should be easy to actually use once you’ve focused on the element. You should be able to interact with it by keyboard (for example, if it’s a button, can you press **Enter** to initiate the action?). Also use a screen reader to ensure you can initiate the action.
-- **Is it understandable?** If the element can be found and interacted with, it should be clear what this action does. If it’s a button, you should determine if it has visible text that would be clear out of context of the page. If it doesn’t have visible text, you should check if it has an aria-label or accessible name.
+As you develop your product, it can be helpful to keep accessibility top of mind, by asking yourself a few questions: 
 
+**1. Can all users discover and perceive elements?**
 
-## What products should address
+If you can see or click on it, then it should be discoverable and perceivable by all users, including those using a keyboard or screen reader.
 
-Keep in mind that while PatternFly provides accessible components, we can't guarantee that your products will be accessible. There are a few items outside the scope of PatternFly, so you should keep these items in mind to ensure accessibility in anything you build: 
-<br />
+**2. Can all users interact with elements?**
 
+It should be easy for users to use an element once it's in focus. They should be able to interact with elements and initiate actions via keyboard (for example, pressing **Enter** to initiate a button action) and screen readers.
 
-<AccessibilityDatalist />
+**3. Can all users understand elements?**
+
+It should be clear what actions elements can perform. For example, buttons should have visible text that would be clear out of context of the page. If not, it should have an aria-label or accessible name.
+
+## Extra steps for accessibility 
+
+PatternFly provides accessible components, but **we can't guarantee that your products will be accessible**. There are items outside the scope of PatternFly, so you should keep these in mind to ensure accessibility: 
 
 <br />
 
-## Techniques
-The [WCAG 2.0 techniques](https://www.w3.org/TR/WCAG20-TECHS/Overview.html#contents) provide examples on how to meet accessibility guidelines. The techniques listed below are adopted as standard within PatternFly for handling specific patterns.
+<AccessibilityDatalist />
 
+<br />
 
-### Labeling with Aria
-For sighted users, the context and visual appearance of an element can provide sufficient cues to determine the purpose. But in some situations, this isn’t the case. For example, the X often used in the top-right corner of pop-up divs (light boxes) to indicate the control for closing the div might not be clear to those using assistive technology. 
+## Recommendations for UI elements
```

Comment:
Would this section make sense as a sub-section of the previous one? Like:

```
## Prioritizing accessibility during development
## Extra steps for accessibility
### Recommendations for UI elements
#### Labeling with ARIA and so on...
```

It might also just be a case of  what should be placed in that table vs this "Recommendations..." section. I may also be feeling @jessiehuff comment above about whether this section would come across to a consumer as "It's recommended so it's optional" or something 😆